solutions/silver/cf-1338A.mdx

```diff
@@ -65,6 +65,36 @@ int main() {
 ```
 
 </CPPSection>
+
+<JavaSection>
+```java
+import java.util.*;
+
+public class PoweredAddition {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		int t = sc.nextInt();
+		while (t-- > 0) {
+			int n = sc.nextInt();
+			int Arr[] = new int[n];
+			for (int i = 0; i < n; i++) { Arr[i] = sc.nextInt(); }
+
+			int prev = Arr[0];
+			int res = 0;
```

Comment:
yeah uh can you make this consistent w/ the c++ version
honestly i feel like this is better suited for a user solution

Reply:
okay